### PR TITLE
Registration - Subnet-register

### PR DESCRIPTION
**Pull request by allmightyspiff**
Adds `slcli reg subnet-register`

image

### CODE REVIEW

**Review by ATGE — COMMENTED** [review]
## Testing 

- ✔️ `slcli registration subnet-register 432263 1853956` 

- ✔️ `slcli registration show`, just  I don't sure if the number of subnets displayed has the same displayed in https://cloud.ibm.com/classic/network/rir, because in this case, slcli is not showing all. (27 subnets vs 35 in the portal)

File: SoftLayer/fixtures/SoftLayer_Network_Subnet_Registration.py [hunk 1]
```diff
@@ -64,3 +64,65 @@
...
+                "id": 1,
+                "keyName": "NETWORK",
+                "name": "Network"
+            }
+        },
+        "detailId": 1672055,
+        "id": 2971613,
+        "modifyDate": "",
+        "registration": {
+            "accountId": 307608,
```
[the remark this answers is not in the document]
Comment (allmightyspiff) [reply SoftLayer/fixtures/SoftLayer_Network_Subnet_Registration.py:115]: Looks like the portal calls `SoftLayer_Account::getPublicSubnets`, as there are some subnets that don't have a vlan->networkSpace information on our account. I'll change over to that API to get the same results.

**Comment by ATGE** [comment]
Testing
✔️ slcli registration subnet-register 432263 51990
✔️ slcli registration show
LGTM 🎉

**Comment by FernandoOjeda** [comment]
It is working as expected and Its looks good for me.

**Review by FernandoOjeda — APPROVED** [review]

**Pull request merged**